frontend/src/components/VSearchResultsGrid/VImageGrid.vue

```diff
@@ -14,9 +14,6 @@
         :related-to="relatedTo"
       />
     </ol>
```

Comment:
When there is an error, `VImageGrid` is not displayed at all, so this was unused.